README-SL.md

```diff
@@ -180,4 +177,4 @@ Povezave, ki neposredno usmerjajo na primerne `issues`, s katerim lahko prispeva
 
 ## Licenca
 
-<a rel="license" href="https://creativecommons.org/licenses/by-sa/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://licensebuttons.net/l/by-sa/4.0/88x31.png" /></a><br />This work is licensed under a <a rel="license" href="https://creativecommons.org/licenses/by-sa/4.0/">Creative Commons Attribution-ShareAlike 4.0 International License</a>.
```

Comment:
This can be kept and added back in.

Reply:
https://github.com/freeCodeCamp/how-to-contribute-to-open-source/commit/73dee199115c07bd01eebf021e85d48d048ab98c done